modular_zubbers/code/modules/cargo/packs/security.dm

```diff
@@ -42,7 +42,7 @@
 	crate_name = "security medic crate"
 	desc = "Contains a medical technician kit."
 	access = ACCESS_SECURITY
-	cost = CARGO_CRATE_VALUE * 5.5
+	cost = CARGO_CRATE_VALUE * 11.75 //BUBBER EDIT: DOUBLES MEDICAL KIT COSTS
```

Comment:
This is only purchased as a departmental order to begin with???
Why?

Reply:
It's the same one as in cargo imports.